sources/packages/backend/apps/api/src/route-controllers/student/student.institutions.controller.ts

```diff
@@ -68,4 +70,26 @@ export class StudentInstitutionsController extends BaseController {
   ): Promise<StudentProfileAPIOutDTO> {
     return this.studentControllerService.getStudentProfile(studentId);
   }
+
+  /**
+   * This controller returns all student documents uploaded by student uploader.
```

Comment:
I know the comments may be copied over from other controller. I would suggest the comment to be something like `Get all the documents uploaded by student.`